scripts/Frequency_Domain_Analysis.py

```diff
@@ -0,0 +1,29 @@
+#pylint: disable=invalid-name
+from __future__ import (absolute_import, division, print_function)
+from Muon import FFT_presenter
```

Comment:
I was harsh with Anton on this so it seems fair here too. I'd like our Python files to start having a bit more of a standard layout w.r.t to the imports at least. The idea is to follow this style:

```
# List all imports, sorted within each section (stdlib/third-party (including mantid)/local muon imports).
# For 'import foo', use one import per line.  For 'from foo.bar import a, b, c'
# it's OK to import multiple items, use the parenthesized syntax 'from foo
# import (a, b, ...)' if the list needs multiple lines.
# Separate stdlib, third-party, and Muon imports by a blank line.
```

See Anton's recent changes: https://github.com/mantidproject/mantid/pull/20229/commits/38bd0545084d3c0d30a62f18dbbf59313c579eaa